recipe/meta.yaml

```diff
@@ -1,6 +1,15 @@
 {% set version = "3.3.2" %}
 {% set build = 1 %}
 
+# conda build is not making unique hashes for the different
+# fortran compiler versions
+# we are hacking the build number to do that for now
+{% if fortran_compiler_version == "9" %}
+{% set build = build*2 + 0 %}
+{% else %}
+{% set build = build*2 + 1 %}
+{% endif %}
+
```

Comment:
@isuruf I decided to hack the build number. Easy to take out and pretty harmless.

Reply:
@isuruf Is the root cause of the issue ever going to be fixed? Who is taking responsibility for this hack and any future consequences? Once the root problem is fixed, the hack should be reverted, but then if this recipe is ever rebuilt, the `build` number should be increased from 1 to 4. Good luck remembering that little detail.

Reply:
I am taking responsibility for this. Also the workaround uses c indexing so bumping the build number by one at the top of the recipe will work fine.

Reply:
@beckermr My point is, once you remove this hack, we should bump the build number from the current ` build = 1`  to  `build = 4`, otherwise the build numbers for the new builds may conflict with the previous ones you generated in this PR. Perhaps I'm missing something, in that case just ignore my comments. It is just that it makes me quite nervous to be a maintainer of a package, and being almost always in the dark with the quirks of the system. Sorry if I sounded rude, that was not the intention.

Reply:
Yep, I understand. You should always feel free to bump me on this package and I am happy to add myself as a maintainer as well. We have an issue open on conda-build for this quirk. There is also no harm in keeping this hack around until the next version bump even if we remove it later.

Reply:
Actually, I am already a maintainer here, so do feel free to bump me if you have questions or are unsure.